tasks/sep/004-fpac-upgrade/.env

```diff
@@ -0,0 +1,3 @@
+ETH_RPC_URL="https://ethereum-sepolia.publicnode.com"
+PROXY_ADMIN_OWNER="0x4377BB0F0103992b31eC12b4d796a8687B8dC8E9"
+SAFE_NONCE="12"
```

Comment:
It does work that way, you can leave it as `SAFE_NONE=` and it will fetch the latest nonce: https://github.com/base-org/contracts/blob/3a566fcc0e01535cf93dbcef6704640cbde222aa/script/universal/MultisigBase.sol#L27-L36